image_writers_guide/sti.adoc

```diff
@@ -10,70 +10,52 @@
 toc::[]
 
 == Overview
+The main advantage of using STI for building reproducible Docker image is the ease of use for developers. As a builder image author, you must be aware of the two basic requirements for the best possible STI performance: the required image contents and STI scripts.
 
-The main advantage of using sti for building reproducible docker image ease of use
-for developer. To meet that criteria you, as a builder image author, should be aware
-of the two basic requirements for the best possible sti performance, these are:
+== Required Image Contents
+The build process consists of three fundamental elements which are combined into a final Docker image:
 
-* link:[#Required image contents]
-* link:[#STI scripts]
+- sources
+- STI scripts
+- builder image
 
+During the build process, STI must place sources and scripts inside the builder image. To do so, STI creates a tar file containing the two, then streams that file into the builder image. Before executing the `sti assemble` script, STI untars that file and places its contents into the location specified with the `--location` flag or the `STI_LOCATION` environment variable from the builder image (the default location is [filename]#/tmp#).
```

Comment:
[Suggestion]
I think we should try to eliminate any ambiguity with the terms we use. For example, I think it's best to state what exactly "containing the two" means. In this case, it would be better to say "....creates a tar file that contains the sources and the scripts, then streams.....".

"......from the builder image (the default location is [filename]#/tmp#)." ==>> "......from the builder image, with the default location being [filename]#/tmp#)."